stdlib/crypto/src/main/ballerina/src/crypto/Module.md

```diff
@@ -115,23 +113,23 @@ public function main() returns error? {
          password: "ballerina"
      };
      crypto:PrivateKey privateKey = 
-        check crypto:decodePrivateKey(keyStore, "ballerina", "ballerina");
+        checkpanic crypto:decodePrivateKey(keyStore, "ballerina", "ballerina");
```

Comment:
Shouldn't we remove these samples from Module.md and point to the BBE? Crypto BBE contains all these functions.
https://v0-991.ballerina.io/learn/by-example/crypto.html